### PR TITLE
Topdown map matplotlib display utility

### PR DESCRIPTION
**Pull request by aclegg3**
## Motivation and Context
Provide a convenience utility to display a topdown map via matplotlib.

## How Has This Been Tested

Local testing in Colab. Example output:

Default:
image

With key-points and title:
image

With trajectory and title:
image

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: habitat_sim/utils/viz_utils.py [hunk 1]
```diff
@@ -94,3 +96,55 @@ def display_video(video_file: str, height: int = 400):
         else:
             opener = "open" if sys.platform == "darwin" else "xdg-open"
             subprocess.call([opener, video_file])
+
+
+def display_topdown_map(
+    topdown_map, plot_title=None, trajectory=None, key_points=None, marker_alpha=0.8
```
Comment (Skylion007) [inline habitat_sim/utils/viz_utils.py:102]: May want some type annotations here. Otherwise looks good!

**Comment by erikwijmans** [comment]
We should make sure to not duplicate the functionality here: https://github.com/facebookresearch/habitat-api/blob/master/habitat/utils/visualizations/maps.py too much

**Comment by aclegg3** [comment]
> We should make sure to not duplicate the functionality here: https://github.com/facebookresearch/habitat-api/blob/master/habitat/utils/visualizations/maps.py too much

Thanks for pointing this out. We've been wanting to display topdown map visualizations on the sim side in a simple way using the existing C++ Pathfinder topdown map implementation. This module looks much more configurable, but would need to be converted for use in Habitat-sim.

**Comment by Skylion007** [comment]
> > We should make sure to not duplicate the functionality here: https://github.com/facebookresearch/habitat-api/blob/master/habitat/utils/visualizations/maps.py too much
> 
> Thanks for pointing this out. We've been wanting to display topdown map visualizations on the sim side in a simple way using the existing C++ Pathfinder topdown map implementation. This module looks much more configurable, but would need to be converted for use in Habitat-sim.

This is a common theme and will lead to duplicate in Habitat-API that will only grow over time, no? We can move things out of habitat-sim to Habitat-API, but then we may need those utilities later without wanting to wrap around Habitat-API. We should keep this in mind for future Habitat-API design.

**Comment by aclegg3** [comment]
Using the API utilities.

**Pull request closed without merging**